src/chrome/content/rules/FreeDesktop.xml

```diff
@@ -51,24 +47,10 @@
 	<target host="xcb.freedesktop.org" />
 	<target host="xorg.freedesktop.org" />
 
-	<!--	Complications:
-				-->
-	<target host="download.freedesktop.org" />
-	<target host="sitewranglers.freedesktop.org" />
-
-		<test url="http://people.freedesktop.org/~airlied/" />
```

Comment:
Please keep this test url because there is no content on `http://people.freedesktop.org/`.